js/functions.js

```diff
@@ -0,0 +1,22 @@
+const isShorterThan = (string, length) => string.length <= length;
+
+const isPalindrome = function(string) {
+  const palindrome = string.toLowerCase().replaceAll(' ', '');
+  for (let i = 0; i < Math.floor(palindrome.length / 2); i++) {
+    if (palindrome.at(i) !== palindrome.at((palindrome.length - 1) - i)) {
```

Comment:
> at(i)

Хайпово-хайпово :stuck_out_tongue_winking_eye:  но ты же тогда можешь так делать
```suggestion
    if (palindrome.at(i) !== palindrome.at(-i)) {
```

Reply:
блин ))

Reply:
@DaniilFomin чуть перепутал тоже
```suggestion
    if (palindrome.at(i) !== palindrome.at(-1 - i)) {
```

Reply:
он проверяет в цикле с at 
0 : -0
1 : -1
И так далее 
таким образом он отстаёт на единицу индекса с конца